README.md

```diff
@@ -70,6 +70,12 @@ This action will load changed components into Code Pipeline on the mainframe fro
 | `showEnv` | **Optional** | Show value of environment variables for debugging. Possible values are `true` or `false` |
 | `assignmentPrefix` | **Optional** | The prefix to be used for the Code Pipeline container. For example, `"PLAY"` |
 | `ispwConfigPath` | **Optional** | The path to Code Pipeline YAML configuration. For example, `"demo\ispwconfig.yml"` |
+| `gitCommit` | **Optional** | The Git commit hash (long or short) or a colon-delimited list of file paths in the workspace. This input parameter needs to be used in combination with input parameter gitFromHash |
```

Comment:
'or a colon-delimited list of file paths in the workspace.' this statement is already a part of 'gitCommitFile' parameter. I suggest to remove it from here unless it has some other significance.